tools/pubsys/src/aws/validate_ami/results.rs

```diff
@@ -0,0 +1,714 @@
+//! The results module owns the reporting of EC2 image validation results.
+
+use super::ami::{ImageDef, Result};
+use aws_sdk_ec2::Region;
+use serde::{Deserialize, Serialize};
+use std::collections::{HashMap, HashSet};
+use std::fmt::{self, Display};
+use std::str::FromStr;
+use tabled::{Table, Tabled};
+
+/// Represent the possible status of an EC2 image validation
+#[derive(Debug, Eq, Hash, PartialEq, Serialize, Deserialize)]
+pub(crate) enum AmiValidationResultStatus {
+    /// The expected value was equal to the actual value
```

Comment:
In the case of AMIs can we be more specific than `value`? In this case do we actually mean:

```rust
/// The AMI was found and is public
```